core/os/input_event.cpp

```diff
@@ -34,8 +34,52 @@
  */
 
 bool InputEvent::operator==(const InputEvent &p_event) const {
+	if (type != p_event.type){
+		return false;
+	}
 
-	return true;
+	switch(type) {
```

Comment:
It would have been good to mention in the commit log that you're doing more than "showing the last added action on InputMap", since this code chunk effectively fixes broken InputEvent comparison, which is relevant beyond this usability enhancement :)